pkgs/dart_services/lib/src/common.dart

```diff
@@ -21,30 +21,17 @@ import 'dart:ui_web' as ui_web;
 
 import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
-import 'generated_plugin_registrant.dart';
+import 'generated_plugin_registrant.dart' as pluginRegistrant;
 import 'main.dart' as entrypoint;
 
 Future<void> main() async {
-  registerPlugins(webPluginRegistrar);
   await ui_web.bootstrapEngine(
     runApp: () {
-      return entrypoint.main();
+      entrypoint.main();
```

Comment:
👍

Reply:
What does this change do?

Reply:
@johnpryan fixes #2966

Reply:
Got it... We might not need to maintain this code any more. The comment above is old and out of date since https://github.com/flutter/flutter/pull/144434 landed and we integrated the new bootstrapping (https://github.com/dart-lang/dart-pad/pull/2960)